WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -631,7 +632,12 @@ protected void signInError(int messageId, String clientResponse) {
         AnalyticsTracker.track(Stat.LOGIN_FAILED);
         FragmentTransaction ft = getFragmentManager().beginTransaction();
         SignInDialogFragment nuxAlert;
-        if (messageId == org.wordpress.android.R.string.account_two_step_auth_enabled) {
+        if (!NetworkUtils.isNetworkAvailable(getActivity())) {
```

Comment:
Just noticed that we're basically "eating" all the network connection errors returned by Volley during login:
See [LoginWPCom.java#L26](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2270-no-connection-login-screen/WordPress/src/main/java/org/wordpress/android/ui/accounts/helpers/LoginWPCom.java#L26). 
What will happen in case of Network timeout for example? 

The condition ` if (!NetworkUtils.isNetworkAvailable(getActivity()))` is not true, and the app will show the classic "we cannot log you in" error message.